src/olympia/blocklist/cron.py

```diff
@@ -23,10 +24,7 @@ def upload_mlbf_to_kinto():
         log.info('Upload MLBF to kinto cron job disabled.')
         return
     last_generation_time = get_config(MLBF_TIME_CONFIG_KEY, 0, json_value=True)
-    if last_generation_time > _get_blocklist_last_modified_time():
-        log.info(
-            'No new/modified Blocks in database; skipping MLBF generation')
-        return
+    last_blocked_count = get_config(MLBF_COUNT_CONFIG_KEY, 0, json_value=True)
 
     log.info('Starting Upload MLBF to kinto cron job.')
```

Comment:
Would it make sense to move this `log` statement below the new `if` check, so that it doesn't appear in the log if we choose not to upload?

Reply:
we could, but I left it because that's the point when we define `generation_time` (and we start doing database queries)